### DISCUSSION TITLE
Show alternative (shorhand) names in css popup in vscode

### DISCUSSION THREAD

**Discussion by kuchta** [master-co/css#315] [Ideas]
Being able to write standard CSS properties is something that's truely needed, because who wants to consult another documentation (as is the case in Tailwind e.g.) for the mapping. One thing that I'm missing though is seeing alternative property shorthand in the IDE so that I can first write full css rule and then see if I can make it more concise.

BTW, thank's for great work. This is something really needed 🙏🏻❤️

**Comment by 1aron** [comment]
Coincidentally, the next RC contains this improvement. 😉

**Comment by 1aron** [comment]
🚀 Released https://marketplace.visualstudio.com/items?itemName=masterco.master-css-vscode
